varats/utils/project_util.py

```diff
@@ -72,3 +72,16 @@ def get_tagged_commits(project_name: str) -> tp.List[tp.Tuple[str, str]]:
             for ref_split in [ref.strip().split() for ref in ref_list]
         ]
         return refs
+
+
+def get_all_revisions_between(a: str, b: str) -> tp.List[str]:
```

Comment:
```suggestion
def get_all_revisions_between(c_start: str, c_end: str) -> tp.List[str]:
```
var names could be a bit more precise :smile: `c_start` or `start_commit`... etc